time-tracker-webapi/test/TimeTracker.Library.Test/Services/AdminReportServiceTest.cs

```diff
@@ -7,9 +14,38 @@ namespace TimeTracker.Library.Test.Services
     public class AdminReportServiceTest
     {
         [Fact]
-        public async Task GetAllUsers_returnsExpectedUsers()
+        public async Task GetAllUsersReport_returnsExpectedUsers_withHours()
         {
-            // todo:
+            var options = TestHelpers.BuildInMemoryDatabaseOptions("adminReport");
+
+            using (var context = new TimeTrackerDbContext(options))
+            {
+                TestHelpers.AddClientAndProject(context);
+                var testUsers = TestHelpers.AddTestUsers(context);
+
+                TimeEntryService timeEntryService = new TimeEntryService(testUsers.First().UserId, context);
+                DateTime currentDate = DateTime.UtcNow.Date;
+                DateTime date = new DateTime(currentDate.Year, 2, 15);
+                
+                await timeEntryService.CreateBillableTimeEntry(date, 8, 1, 1);
```

Comment:
Depending on its usage, the method signature for Create(Non)BillableTimeEntry might benefit from taking in a TimeEntry object instead of a bunch of parameters.
This might improve the readability when consuming and potentially reduce refactoring mistakes.

Reply:
hmm yeah, or a specific POJO just for creating a billable entry - since the TimeEntry object has more parameters than needed to create a billable entry.